src/Microsoft.SqlTools.Serialization/Microsoft.SqlTools.Serialization.csproj

```diff
@@ -0,0 +1,21 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>netcoreapp2.0</TargetFramework>
+    <PackageId>MicrosoftSqlToolsSerialization</PackageId>
```

Comment:
As this is an exe, lets consider using simpler names. `SqlSerializationService` may be better. We want to align on better names for all our exes in the future